include/graphics/nxwidgets/itextbox.hxx

```diff
@@ -208,7 +208,7 @@ namespace NXWidgets
      * @return position The cursor position.
      */
 
-    virtual const int getCursorPosition(void) const = 0;
+    virtual const int32_t getCursorPosition(void) const = 0;
```

Comment:
should we change https://github.com/apache/incubator-nuttx-apps/blob/master/include/graphics/nxwidgets/cscrollingtextbox.hxx#L329 to int instead?
moveCursorToPosition at https://github.com/apache/incubator-nuttx-apps/blob/master/include/graphics/nxwidgets/cscrollingtextbox.hxx#L320 also need to change.

Reply:
i dunno. i'm not familiar with this piece of code.
is there any reason to prefer int over int32_t here?

Reply:
two reason:

1. Other function(e.g. moveCursorToPosition) use int
2. Only CScrollingTextBox use int32_t, all other place use int

Reply:
ok, i will fix

Reply:
done